packages/client/mutations/RemoveOrgUserMutation.ts

```diff
@@ -92,19 +102,25 @@ const mutation = graphql`
   }
 `
 
-export const removeOrgUserOrganizationUpdater = (payload, {atmosphere, store}) => {
+export const removeOrgUserOrganizationUpdater: SharedUpdater<RemoveOrgUserMutation_organization> = (
+  payload,
+  {atmosphere, store}
+) => {
   const {viewerId} = atmosphere
-  const removedUserId = getInProxy(payload, 'user', 'id')
-  const removedOrgUserId = getInProxy(payload, 'organizationUserId')
-  const orgId = getInProxy(payload, 'organization', 'id')
+  const removedUserId = payload.getLinkedRecord('user').getValue('id')
```

Comment:
`getInProxy` was an old helper i made a loong time ago, before I PRed relay to add strong types & before the `?` existed in javascript. Now that we have those, it's cleaner to use the native methods